doc/site/content/en/docs/Contribution guidelines/dev_new_stages.md

```diff
@@ -7,7 +7,93 @@ categories: [Examples]
 tags: [test, sample, docs]
 ---
 
-## Base Classes
+### Introduction
+This guide covers the basics of how to develop a new stage for Lucille and what some of its required and optional components are. After reading this, you should be able to start development with a good foundation on how testing works, how configuration is handled, and what features the Stage class affords us. 
 
+**Note: Angle brackets (<…>) are used to show placeholder information that is meant to be replaced by you when using the code snippets.**
+### Prerequisite(s)
+- An up-to-date version of Lucille that has been appropriately installed 
 
-## Unit Tests
\ No newline at end of file
+- Please refer to **Quick Start Guides - Lucille Local Mode** for more info on how to setup Lucille

```

Comment:
Just noting that the next step of documentation should be adding this so we can get these properly linked. I don't view it as blocking this PR.